### DISCUSSION TITLE
feat: Adding Next/Prev buttons on Tweet detail drawer pane.

### DISCUSSION THREAD

**Discussion by ogunbaysal** [acikkaynak/deprem-yardim-frontend#550] [Ideas]
## Task Definition
We might provide a faster navigation by adding `next` and `previous` marker options on the marker detail section.
In the case of app being used on a mobile device, it will be a smoother option while navigating among tweets for the same location.

## Other Evaluated Alternatives
None

## Final Notes

![Screenshot_20230209_002004](https://user-images.githubusercontent.com/33205381/217653697-216a5565-3040-496e-8aeb-7bdcd41f839c.png)

**Comment by cdagli** [comment]
We gotta get a review from #tasarim-ux-ui channel folks, a different drawer is being displayed on a mobile device.

**Comment by hasanugr** [comment]
In order to handle this work by frontend team, the data delivered by backend service should be sorted by the distance.
Before we tackle can we issue a task to backend team for sorting?

**Comment by 0xCakin** [comment]
There are high-priority items but gathering more info about this. We might not be able to include it in the next release.
